app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express")
+const app = express()
+app.use(express.json())
+
+const {getTopics} = require("./controllers/topics.controllers")
```

Comment:
To tidy this up a bit, I'd encourage you to have all your `require` statements together in one place, at the top of the file. So moving this to line 1 or 2 would neaten the code up slightly.